examples/instantsearch/package.json

```
@@ -19,7 +19,7 @@
     "preact": "10.5.13"
   },
   "devDependencies": {
-    "parcel": "2.5.0"
+    "parcel": "2.7.0"
```

The version bump seems unnecessary, as it serves and builds well with the other adjustments. 

```suggestion
    "parcel": "2.5.0"
```

(We also need to rerun `yarn install` in order to clean up **yarn.lock**)

---

why not the latest one?

---

I was initially looking at maintaining a single version of parcel, but instead of locking to 2.5.0 we should do the opposite and bumping eventually all of our samples to the latest. So it's good for me in the end here.